components/bl-social-login-button-component/src/button.js

```diff
@@ -0,0 +1,46 @@
+import { IconsMap } from './icons-map';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export function Button({ socialNet, showButtonIcon, onClick }) {
+  const handleClick = async () => {
+    async function socialLogin(providerCode, fieldsMappings, scope, options, redirectToPage, callbackUrlDomain) {
+      if (BackendlessUI.DeviceAPI.isMobile()) {
+        const user = await BackendlessUI.DeviceAPI
+          .socialLogin(providerCode, fieldsMappings, scope, options, callbackUrlDomain)
+
+        Backendless.UserService.setCurrentUser(user, true)
+
+        if (redirectToPage) {
+          BackendlessUI.Navigator.goToPage(redirectToPage)
+        }
+      } else {
+        const redirectAfterLoginUrl = redirectToPage
+          ? window.location.href.split('?')[0] + `?page=${ redirectToPage }`
+          : window.location.href
+
+        window.location = await Backendless.UserService
+          .getAuthorizationUrlLink(providerCode, fieldsMappings, scope, false, redirectAfterLoginUrl, callbackUrlDomain)
+      }
+    }
+
+    await socialLogin(socialNet, null, null, null, null, null)
```

Comment:
Please ask @Valodya if we must add some properties which could be accepted by Social Login at SDK. I a'm about `fieldsMappings, scope, options, redirectToPage, callbackUrlDomain`

Reply:
I assume it would be helpful

Reply:
I added these options and got an error (fieldsMappings is not defined)